charts/kubeclarity/templates/deployment.yaml

```diff
@@ -50,8 +42,29 @@ spec:
       initContainers:
         - name: '{{ include "kubeclarity.name" . }}-wait-for-pg-db'
           image: {{ index .Values "kubeclarity-postgresql" "image" "registry" }}/{{ index .Values "kubeclarity-postgresql" "image" "repository" }}:{{ index .Values "kubeclarity-postgresql" "image" "tag" }}
-          command: ['sh', '-c', 'until pg_isready -h {{ $dbHost }} -p {{ $dbPort }} -U {{ $dbUser | quote }} -d "dbname={{ $dbName }}";
+          command: ['sh', '-c', 'until pg_isready -h $DB_HOST -p $DB_PORT_NUMBER -U $DB_USER -d "dbname=$DB_NAME";
```

Comment:
it would probably be safer to make sure that these variables are quoted:

```suggestion
          command: ['sh', '-c', 'until pg_isready -h "$DB_HOST" -p "$DB_PORT_NUMBER" -U "$DB_USER" -d "dbname=$DB_NAME";
```

otherwise its possible someone could set the username to a something like: `foo; echo /etc/passwd` and perform command injection.